_docs/cluster/manual/slate-worker-node.md

```diff
@@ -10,16 +10,98 @@ type: markdown
 
 To distribute work assigned to a SLATE cluster, worker nodes can be networked to a SLATE Master node.
 
-All SLATE Worker nodes should be set up using the "Setting Up a SLATE Node" instructions above.
+## Requirements
 
-### KubeADM Join
+Prepare the Worker node by following the steps described in:
+1. [Operating System Requirements](/docs/cluster/manual/operating-system-requirements.html)
+2. [Install Containerd](/docs/cluster/manual/containerd.html)
+3. [Install Kubernetes](/docs/cluster/manual/kubernetes.html)
 
-On your Master node, run the following command to get a full join command for the Master's cluster:
+## Gather Cluster Information
 
+SSH into your SLATE Master node and gather the following information using the commands described below:
+
+| Name | Command |
+| --- | --- |
+| Kubernetes Token | `$(kubeadm token create)` |
+| Kubernetes Discovery Certificate Hash | `$(openssl x509 -pubkey -in /etc/kubernetes/pki/ca.crt | openssl rsa -pubin -outform der 2>/dev/null | openssl dgst -sha256 -hex | sed 's/^.* //')` |
+
+## Define Variables
+
+SSH into your SLATE Worker node and define the following variables:
+
+1. The token and certificate from the steps above:
+
+   ```shell
+   K8_TOKEN=<value>
+   K8_DISCO_CERT=<value>
+   ```
+   {:data-add-copy-button='true'}
+
+2. IPv4 and IPv6 addresses for both nodes. For example:
+
+   ```shell
+   # master node
+   MASTER_IPV4_ADDR=192.168.0.3
+   MASTER_IPV6_ADDR=2001:db8:1234:5678::1
+   # worker node
+   WORKER_IPV4_ADDR=192.168.0.4
+   WORKER_IPV6_ADDR=2001:db8:1234:5678::2
+   ```
+   {:data-add-copy-button='true'}
+
+## Build Kubeadm Join
```

Comment:
Why is this needed? Strictly for IPv6?

Reply:
I have added comments to the following to clarify in `slate-worker-node.md`:
```shell
 # master node
   MASTER_IPV4_ADDR=203.0.113.3               # required if the control-plane is IPv4
   MASTER_IPV6_ADDR=2001:db8:1234:5678::1     # required if the control-plane is IPv6
```
These variables are used further down the doc page in either the IPv4 or IPv6 join YAML templates. Does that answer your question @LincolnBryant ?